dotCMS/src/main/java/com/dotmarketing/quartz/QuartzUtils.java

```diff
@@ -502,11 +444,10 @@ else if (sched.getTrigger(triggerName, triggerGroup) != null)
 	 * 
 	 */
 	public static boolean removeJob(String jobName, String jobGroup) throws SchedulerException {
-		Scheduler sched = DotSchedulerFactory.getInstance().getSequentialScheduler();
+		Scheduler sched = DotSchedulerFactory.getInstance().getScheduler();
 		boolean result1 = removeJob(jobName, jobGroup, sched);
-		sched = DotSchedulerFactory.getInstance().getScheduler();
-		boolean result2 = removeJob(jobName, jobGroup, sched);
-		return result1 | result2;
+
+		return result1 ;

```

Comment:
Codacy found an issue: [Consider simply returning the value vs storing it in local variable 'result1'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6213816)